packages/libs/components/src/map/SemanticMarkers.tsx

```diff
@@ -48,11 +49,33 @@ export default function SemanticMarkers({
   // react-leaflet v3
   const map = useMap();
 
-  // cancel marker selection with a single click on the map
+  const { enqueueSnackbar } = useSnackbar();
+
+  // detect zooming and panning
+  const [isZoom, setIsZoom] = useState(false);
+  const [isDragend, setIsDragend] = useState(false);
+
   useMapEvents({
+    // cancel marker selection with a single click on the map
     click: () => {
       if (setSelectedMarkers != null) setSelectedMarkers(undefined);
     },
+    // check zooming: zoomend somehow does not work consistently here
+    // but zoomend just worked fine at MapVEuMap
```

Comment:
I think we have a more general issue with trying to handle the same event in different places, see #952 

If we can't reliably handle the same events in different components then I suppose we need a different approach here. One that doesn't rely on Leaflet events.

How do we distinguish between no selected marker IDs are present in `consolidatedMarkers` because
1. we panned them all off the screen
2. we changed the zoom level
?

One approach would be to count the number of characters in `selectedMarkers[0]` and compare it with the number of characters in the first `consolidatedMarkers`'s ID.  The number of characters is equivalent to the zoom level.

Reply:
@bobular Those events (zoom & dragend) are consistent for me, perhaps because they were not used at MapVEuMap's map events. Anyway, your suggestion would also work if we will only distinguish those two cases (zooming and panning) 👍

Reply:
@bobular Yes, Sir! 😄 Usually I am working for VEuPathDB on Monday and Tuesday so I didn't have a plan to do that this week anyway 😉